ui/i18n/datepicker-bg.js

```diff
@@ -13,24 +13,74 @@
 }( function( datepicker ) {
 
 datepicker.regional.bg = {
-	closeText: "затвори",
-	prevText: "&#x3C;назад",
-	nextText: "напред&#x3E;",
+	closeText: "Затвори",
+	prevText: "&#x3C;Назад",
+	nextText: "Напред&#x3E;",
 	nextBigText: "&#x3E;&#x3E;",
-	currentText: "днес",
-	monthNames: [ "Януари","Февруари","Март","Април","Май","Юни",
-	"Юли","Август","Септември","Октомври","Ноември","Декември" ],
-	monthNamesShort: [ "Яну","Фев","Мар","Апр","Май","Юни",
-	"Юли","Авг","Сеп","Окт","Нов","Дек" ],
-	dayNames: [ "Неделя","Понеделник","Вторник","Сряда","Четвъртък","Петък","Събота" ],
-	dayNamesShort: [ "Нед","Пон","Вто","Сря","Чет","Пет","Съб" ],
-	dayNamesMin: [ "Не","По","Вт","Ср","Че","Пе","Съ" ],
+	currentText: "Днес",
+	monthNames: [
```

Comment:
I agree this is actually the nicer CGL style, but I would like to keep the code style as it was before. Reasons are: we want to keep consistency and it makes the review harder. Please change to one line per localization key.